### PR TITLE
Update value-transfer.md

### PR DESCRIPTION
**Pull request by Phyloiota**
starting content

# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancment

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

**Comment by Phyloiota** [comment]
also here, can be published and extended, improved

File: docs/learn/about-iota/value-transfer.md [hunk 1]
```diff
@@ -1,34 +1,108 @@
...
 
-----
+In this example, Alice had 10 i that she transferred to Bob.
+Therefore, the Tangle now includes a value message that transferred those 10 i to Bob.
 
-<br/>
+### Choosing where to attach messages
 
-**need to add a footer section for further content - (three cards)**
+To transfer 10 i to Bob, Charlie must attach a value message to the Tangle. To do so, he needs between 1 and 8 tip messages that he can attach his own message.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:33]:
```suggestion
To transfer 10 i to Bob, Charlie must attach a value message to the Tangle. For that, he needs between 1 and 8 tip messages that he can attach to his own message.
```

To, To, To :trollface:

File: docs/learn/about-iota/value-transfer.md [hunk 2]
```diff
@@ -1,34 +1,108 @@
...
+
+To get these tip messages, Charlie requests them from a node.
+
+The node selects this messages by using an algorithm called uniform random tip selection (URTS). This algorithm selects between 1 and 8 valid tip messages that lead to a valid ledger state.
+
+* The ledger state is a record of all the value message that have transferred IOTA tokens in the Tangle, including the state of all balances on addresses. This state must always be balanced: The total number of IOTA tokens must always add up to thetotal supply.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:39]:
```suggestion
* The ledger state is a record of all the value message that have transferred IOTA tokens in the Tangle, including the state of all balances on addresses. This state must always be balanced: The total number of IOTA tokens must always add up to the total supply.
```

File: docs/learn/about-iota/value-transfer.md [hunk 3]
```diff
@@ -1,34 +1,108 @@
...
+
+### Waiting for the messages to be confirmed
+
+Anyone is free to send messages to a node at any time. As a result, it is not always a simple matter of selecting valid tip messages. The milestone, however, determines which messages are included and confirmed based on deterministic ordering.
+
+For example, what if Alice actually attached two messages to the Tangle: One that transferred 10 i to Bob, and one that transferred 10 i to Charlie? Together, these messages would be called a double spend because they try to transfer the same IOTA tokens to different addresses.
+
+Both of Alice's messages could not be part of the ledger state because it would result in a negative balance for Alice’s address: - 10 i.
+
+Messages in the Tangle are confirmed by special messages called milestones. Those milestones are issued by a special node called Coordinator, that is operated by the IOTA Foundation and is therefore still a centralized part in the Consensus.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:63]:
```suggestion
Messages in the Tangle are confirmed by special messages called milestones. Those milestones are issued by a special node called Coordinator, that is operated by the IOTA Foundation and is therefore still a centralized part of the Consensus.
```

File: docs/learn/about-iota/value-transfer.md [hunk 4]
```diff
@@ -1,34 +1,108 @@
...
+
+After Charlie's message is confirmed by a milestone, the nodes update his and Bob's balances to reflect the transfer of 10 i to Bob's address.
+Bob can now spend his new IOTA tokens.
+
+Lets go a bit more into details and observe the single stepd taken by the participants:
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:68]:
```suggestion
Let's go a bit more into details and observe the single steps taken by the participants:
```

File: docs/learn/about-iota/value-transfer.md [hunk 5]
```diff
@@ -1,34 +1,108 @@
...
+
+
+## Clients
+
+Everything starts with a client. A client is a software that initiates and creates IOTA transactions for a user (humans/machines/devices).
+Most clients for users will come in the form of a wallet software like the 
+
+- [IOTA Firefly Wallet](https:/firefly.iota.org/)
+
+But also simple command-line tools for professional users like the
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:78]:
```suggestion
But also simple command-line tools for professional users, like the 
```

File: docs/learn/about-iota/value-transfer.md [hunk 6]
```diff
@@ -1,34 +1,108 @@
...
+
+- [IOTA Firefly Wallet](https:/firefly.iota.org/)
+
+But also simple command-line tools for professional users like the 
+
+- [CLI Wallet](https:/https://github.com/iotaledger/cli-wallet/) 
+
+are available. 
+
+Machines, sensors, and devices, etc will use the code of the IOTA Wallet Libraries to create and execute the transactions autonomously.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:84]:
```suggestion
Machines, sensors, and devices, etc. will use the code of the IOTA Wallet Libraries to create and execute the transactions autonomously.
```

File: docs/learn/about-iota/value-transfer.md [hunk 7]
```diff
@@ -1,34 +1,108 @@
...
+
+To send a value transaction into the IOTA network, a client creates an IOTA message that includes a "signed transaction payload". In this payload, all the data is specified to tell the IOTA network how many tokens from address A (owned by the issuer of the message) should be transferred to address B and makes it possible to upgrade the state of the IOTA Ledger accordingly.
+
+This signed transaction payload must contain a digital signature that guarantees that the sender of the transaction is the owner of the address where those funds are stored at the moment. This is guaranteed through signing the transaction with the private key of that address. A public key, that is also part of this message will be used to validate the ownership over those funds. Read more about how private and public keys work together to establish this here:
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:88]:
```suggestion
This signed transaction payload must contain a digital signature that guarantees that the sender of the transaction is the owner of the address where those funds are stored at the moment. This is guaranteed through signing the transaction with the private key of that address. A public key, that is also part of this message, will be used to validate the ownership over those funds. Read more about how private and public keys work together to establish this here:
```

File: docs/learn/about-iota/value-transfer.md [hunk 8]
```diff
@@ -1,34 +1,108 @@
...
+
+The next step will be that the client connects to a node in the IOTA network and asks this node for valid tips to include into the message. After the tips (1 - 8 previous already confirmed messages of the tangle) are known to the client, they can be included into the message and therefore create a refference path for this message in the Tangle. Finaly now the client submits this message (that includes the transaction payload) for validation and processing to the node.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:90]:
```suggestion
The next step will be that the client connects to a node in the IOTA network and asks this node for valid tips to include into the message. After the tips (1 - 8 previous already confirmed messages of the tangle) are known to the client, they can be included into the message and therefore create a reference path for this message in the Tangle. Finally, the client now submits this message (that includes the transaction payload) for validation and processing to the node.
```

File: docs/learn/about-iota/value-transfer.md [hunk 9]
```diff
@@ -1,34 +1,108 @@
...
+
+
+## Nodes
+
+Nodes are the bookkeepers, decision-makers, and validators of all information in the IOTA network. Every node in the IOTA network knows the exact status and containing value of all existing addresses in the iota network at a given time. This is called the Ledger State. 
+
+A Node is also the entry point for clients into the network. The clients submit messages to the node over a specified port in the node. The node collects all arriving traffic of this entry port in his inbox and first checks if the message is correctly formatted and can be processed.
+
+If all bits of the message are readable for the node and the node detects the signed transaction payload included in the message a validation process starts.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/value-transfer.md:99]:
```suggestion
If all bits of the message are readable for the node and the node detects the signed transaction payload included in the message, a validation process starts.
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**